releasenotes/notes/Deprecate_legacy_pulse_builder_command-47651b395940e687.yaml

```diff
@@ -0,0 +1,30 @@
+---
+deprecations:
+  - |
+    Deprecated injecting circuit gate operation into the pulse context.
+    
+    Deprecated
+    :func:`qiskit.pulse.builder.call_gate`,
+    :func:`qiskit.pulse.builder.cx`,
+    :func:`qiskit.pulse.builder.u1`,
+    :func:`qiskit.pulse.builder.u2`,
+    :func:`qiskit.pulse.builder.u3`,
+    :func:`qiskit.pulse.builder.x`
+
+    .. code-block:: python
+      # This example shows how to get pulse gate instructions.
+      from qiskit.providers.fake_provider import FakePerth
+      backend=FakePerth()
+      sched=backend.target['x'][(qubit,)].calibration
+    
```

Comment:
Doctests are failing due to `sphinx` error. Following may help you to debug [Code block with line numbers problem](https://groups.google.com/g/sphinx-users/c/3VCVS73nhsg), [What is the point of the Sphinx highlight_language config option if code-block:: doesn't have an optional argument?](https://stackoverflow.com/questions/34739635/what-is-the-point-of-the-sphinx-highlight-language-config-option-if-code-block)

Reply:
Hi! Thanks, @Raghav-Bell 
could you please explain to me how did you find the error?
Please, help me know what tools do i use to find out errors like these?

Reply:
@MozammilQ, Check failed CI logfiles for debugging PR.

Reply:
@Raghav-Bell , I always check the log files, but the failed log file (for `Run Docs build`) showed this:

![Screenshot_20231104_154409](https://github.com/Qiskit/qiskit/assets/54737489/986b6ddb-feef-4953-b33f-8362cdfa52e9)

How did you pin point the error at `.. code-block:: python`
please, help! :) thanks for the reply :)

Reply:
I search for keywords like error or failed. Somewhere in the logfile you may get error printed.

Reply:
![image](https://github.com/Qiskit/qiskit/assets/83136390/c8719fcb-87b3-47d6-bd2b-7ead0cd2bb95)